tests/functional/export_v2/cram/metadata-columns.t

```diff
@@ -0,0 +1,16 @@
+Setup
+
+  $ source "$TESTDIR"/_setup.sh
+
+Run export with tree and metadata with additional columns.
+
+  $ ${AUGUR} export v2 \
+  >  --tree "$TESTDIR/../data/tree.nwk" \
+  >  --metadata "$TESTDIR/../data/dataset1_metadata_with_additional_columns.tsv" \
```

Comment:
Suggestion: These files are small, so define them explicitly in the test.

```
Create files for testing.

  $ cat >metadata.tsv <<~~
  > strain	field_A	field_B
  > tipA	AA	AAA
  > tipB	BB	BBB
  > tipC	CC	CCC
  > tipD	DD	DDD
  > tipE	EE	EEE
  > tipF	FF	FFF
  > ~~

  $ cat >tree.nwk <<~~
  > (tipA:1,(tipB:1,tipC:1)internalBC:2,(tipD:3,tipE:4,tipF:1)internalDEF:5)ROOT:0;
  > ~~

Run export with tree and metadata with additional columns.

  $ ${AUGUR} export v2 \
  >  --tree tree.nwk \
  >  --metadata metadata.tsv \
  …
```

Reply:
Updated in b9b5649914d31786a57c928f9b67781209e43869